src/TerminalLogger.jl

```diff
@@ -169,6 +169,12 @@ function showvalue(io, key, e::Tuple{Exception,Any})
 end
 showvalue(io, key, ex::Exception) = showerror(io, ex)
 
+if VERSION >= v"1.7.0-DEV.1106"
+    @eval begin
+        showvalue(io, key, ex::Base.ExceptionStack) = Base.show_exception_stack(io, ex)
+    end
```

Comment:
I don't think this is necessary, `show()` already works for `ExceptionStack`?

It was intended that we have the small amount of extra decoration of `show()` vs the internal function `show_exception_stack` - an exception stack isn't necessarily an error per se. Eg, you can use

```julia
@info "we ignored a thing"  current_exceptions()
```

BTW if this was kept you don't need the `@eval` here because this is top level code.